mne/decoding/base.py

```diff
@@ -415,14 +415,15 @@ def plot_patterns(self, info, times=None, ch_type=None, layout=None,
                 dict(marker='o', markerfacecolor='w', markeredgecolor='k',
                      linewidth=0, markersize=4)
 
-        outlines : 'head' | dict | None
-            The outlines to be drawn. If 'head', a head scheme will be drawn.
-            If dict, each key refers to a tuple of x and y positions.
-            The values in 'mask_pos' will serve as image mask.
-            If None, nothing will be drawn. Defaults to 'head'.
-            If dict, the 'autoshrink' (bool) field will trigger automated
-            shrinking of the positions due to points outside the outline.
-            Moreover, a matplotlib patch object can be passed for
+        outlines : 'head' | 'skirt' | dict | None
+            The outlines to be drawn. If 'head', the default head scheme will
+            be drawn. If 'skirt' the head scheme will be drawn, but sensors are
+            allowed to be plotted outsize of the head circle. If dict, each key
+            refers to a tuple of x and y positions. The values in 'mask_pos'
+            will serve as image mask. If None, nothing will be drawn.  Defaults
+            to 'head'. If dict, the 'autoshrink' (bool) field will trigger
```

Comment:
You have two "If dict" lines, wouldn't it be a bit more readable to condense them into one? E.g.,

```
The outlines to be drawn. If 'head', the default head scheme will
be drawn. If 'skirt', the head scheme will be drawn, but sensors are
allowed to be plotted outsize of the head circle. If None, nothing 
will be drawn. If dict, each key refers to a tuple of x and y positions
and the 'autoshrink' (bool) field will trigger automated shrinking of
the positions due to points outside the outline. The values in 
'mask_pos' will serve as image mask. Moreover, a matplotlib patch 
object can be passed for advanced masking options, either directly 
or as a function that returns patches (required for multi-axis plots).
 Defaults to 'head'. 
```